contracts/oracles/convex/Cvx3CrvOracle.sol

```diff
@@ -79,7 +79,7 @@ contract Cvx3CrvOracle is IOracle, AccessControl {
         bytes32 base,
         bytes32 quote,
         uint256 baseAmount
-    ) external view virtual override returns (uint256 quoteAmount, uint256 updateTime) {
+    ) external view virtual override returns (uint256, uint256) {
         return _peek(base.b6(), quote.b6(), baseAmount);
```

Comment:
The following allows you to keep the named returns, which are nice for code parsers such as Etherscan's.
```
) external view virtual override returns (uint256 quoteAmount, uint256 updateTime) {
    (quoteAmount, updateTime) = _peek(base.b6(), quote.b6(), baseAmount);
```